index.html

```diff
@@ -114,7 +114,15 @@ <h1>Glossary</h1>
       that handled the chain of requests initiated by this button
       being pressed.
     </dd>
-  </dl>
+    </dl>
+    <dl>
+    <dt><dfn data-lt="opaque">Opaque value</dfn></dt>
+    <dd>
+      An opaque value refers to a value that can only be understood,
+      or processed in any way by the distributed trace participant that generated
+      this value. Any other participant must treat it as a blob of bytes.
```

Comment:
I think the definition is a little too editorial/wordy, and contains the word "must" which may be misconstrued as normative. I think a better definition which assumes less would be this:

```suggestion
      An opaque value refers to a value with unspecified meaning
      or format beyond the limits defined in the specification.
```

Reply:
I agree with your evaluation. Unfortunately PR already merged with this definition. Would you like to submit your wording as a proposal? Or let me know and I'll do it